plugins/source/aws/docs/tables/aws_resiliencehub_app_version_resource_mappings.md

```diff
@@ -18,8 +18,11 @@ This table depends on [aws_resiliencehub_app_versions](aws_resiliencehub_app_ver
 |_cq_parent_id|UUID|
 |account_id (PK)|String|
 |region (PK)|String|
+|app_arn (PK)|String|
```

Comment:
If we are adding `app_arn` doesn't that mean that `account_id` and `region` are duplicated?